hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/slowlog/LogEventHandler.java

```diff
@@ -256,4 +278,32 @@ private boolean isFilterProvided(AdminProtos.SlowLogResponseRequest request) {
     return filteredSlowLogPayloads;
   }
 
+  /**
+   * Poll from queueForSysTable and insert 100 records in hbase:slowlog table in single batch
+   */
+  void addAllLogsToSysTable() {
+    if (queueForSysTable == null) {
+      LOG.warn("hbase.regionserver.slowlog.systable.enabled is turned off. Exiting.");
```

Comment:
If the config is turned off, this will log this warn line in every 10 mins right? Dont think we need to log here at all.